-- src/server/init.server.luau
local GrabHandler      = require(script.GrabHandler)
local DoorService      = require(script.DoorService)
local ToolboxService   = require(script.ToolboxService)
local SofaStateService = require(script.SofaStateService)
local JobService       = require(script.JobService)
local NPCService       = require(script.NPCService)
local ExitService      = require(script.ExitService)

GrabHandler.Init()
DoorService.Init()
ToolboxService.Init()
SofaStateService.Init()
JobService.Init()
NPCService.Init()
ExitService.Init()
